code/modules/psionics/ear.dm

```diff
@@ -0,0 +1,315 @@
+/obj/machinery/eternal_ascension_resonances
+	name = "EGO: E.A.R. T-01-92beta S-23"// EGO as in the ego, T as in Tech, S for Sanity
+	desc = "A grand work of SI's psionic advancements. Marbel pilers that only can hold psionic catalyst, converting the power of many into one."
```

Comment:
```suggestion
	desc = "A grand work of SI's psionic advancements. Marble pillars that only can hold psionic catalysts, converting the power of many into one."
```
"Marble" and "Pillar" should also be corrected throughout the other descriptions